release/src/version-helpers.ts

```diff
@@ -159,7 +159,6 @@ export const getNextVersions = (versionString: string): string[] => {
   // minor releases -> next minor release
   const [major, minor] = versionString
     .replace(/(v1|v0)\./, "")
-    .replace(/.0$/, "")
```

Comment:
I'm guessing the period was meant to be escaped? I can't see why this line was needed though.

Reply:
No tests were broken in the removal of this line 👮

Reply:
Looks good, I think at some point this was handling the fact that our major version milestones are `0.49` and not `0.49.0`, but that logic is now being handled by `getMilestoneName` 👍